packages/migrate/src/utils/byline.ts

```diff
@@ -23,13 +24,13 @@
 import stream from 'stream'
 import util from 'util'
 
-// convinience API
+// convenience API
 export default function byline(readStream, options?: any) {
-  return module.exports.createStream(readStream, options)
+  return createStream(readStream, options)
 }
 
 // basic API
-module.exports.createStream = function (readStream, options) {
+export function createStream(readStream, options) {
```

Comment:
Not related to dependencies update per se, just curious, if there a reason why this code is repeated several times in the different packages?

Reply:
:shrug: Things to be cleaned and shoved into the `@prisma/sdk` :trollface: Also why I started our own utility library since many functional utils are copied across projects.

Reply:
Tech debt topic, something to clean indeed